hack/update_owners.py

```diff
@@ -14,22 +14,23 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+
 import argparse
 import collections
 import csv
 import json
-import os
+from pathlib import Path
```

Comment:
Please import modules not classes

https://github.com/google/styleguide/blob/gh-pages/pyguide.md#22-imports

Reply:
@fejta Ok, will do! :)